src/components/habit-tracker/analytics/month-view/LineChart.tsx

```diff
@@ -39,12 +40,9 @@ interface LineChartProps {
 // Function to generate the ApexCharts options object
 const generateChartOptions = (
   daysInMonth: number,
-  year: number,
-  monthIndex: number
+  month: string
 ): ApexCharts.ApexOptions => {
-  const monthName = new Date(year, monthIndex).toLocaleString("default", {
-    month: "long",
-  });
+  const capitalizedMonth = upperFirst(month);
```

Comment:
no need to create a variable for this. Directly use `upperFirst(month)` on line 102